docs/guides/usage.md

```diff
@@ -0,0 +1,221 @@
+# Execution of pipelines
+
+ZARP consists of three different pipelines. The main pipeline processes the data, the second allows you to download the sequencing libraries from the Sequence Read Archive (SRA), and the third populates a file with the samples and determines sample specific parameters.
+
+If you can create a `samples.tsv` file and fill in the metadata for the different sequencing experiments then the main pipeline can analyze your data.
+
+!!! info "Prerequisites"
+    
+    The code below assume that you have already [installed](./installation.md) ZARP.
+
+## How to run ZARP
```

Comment:
To be consistent with the other headers, add a question mark at the end. Better yet, formulate the header so that it focuses on what the workflow does, just like the other headers do, e.g.:

> How to analyze your RNA-Seq samples?

Reply:
Done